src/components/TopLoader.tsx

```diff
@@ -1,8 +1,11 @@
 'use client'
+import { usePathname } from 'next/navigation'
 import { ProgressLoader } from 'nextjs-progressloader'
 import { Suspense } from 'react'
 
 export const ProgressLoad = () => {
+  const pathname = usePathname()
+  if (pathname.includes('client')) return null
```

Comment:
Do we not need progress loader in client list page at all?

Also as detail route for client is `/detail/:id/cu`, i think we will get progress loader going from detail -> list, but not list -> detail

Reply:
This is an immediate solution as we are approaching release. The `nextjs-progressloader` component has a weird issue where it is sometimes showing loader while clicking the mark done button that's why this is only included in the list page for clients.

Reply:
Got it... since next 13.1 or so we can't listen to route events so fixing it some other way should be tricky